### PR TITLE
NikitK/ Add references on some of the components for cypress automation

### PR DESCRIPTION
**Pull request by NikitK-deriv**
Changes:

-   reference on some of the components and a little bit refactoring

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/partners/affiliate-ib/_dmt5-cards.tsx [hunk 1]
```diff
@@ -215,7 +215,11 @@ const DMT5Cards = ({ data }: DMT5Props) => {
     }
 
     return (
-        <StyledCard padding="2.4rem 2.4rem 0">
+        <StyledCard
+            padding="2.4rem 2.4rem 0"
+            id="affiliate-card
+        "
```
Comment (maryia-binary) [inline src/pages/partners/affiliate-ib/_dmt5-cards.tsx:221]: should this quotation mark hang?🙂

File: src/pages/derivx/_start-derivx.tsx [hunk 2]
```diff
@@ -97,23 +101,27 @@ const demo: RealOrDemoShowType[] = [
 
 const real: RealOrDemoShowType[] = [
     {
+        test_classname_id: 'sign-in',
```
[the remark this answers is not in the document]
Comment (NikitK-deriv) [reply src/pages/derivx/_start-derivx.tsx:104]: Thanks for suggestion and clarify 🙇

File: src/components/elements/accordion.tsx [hunk 3]
```diff
@@ -109,6 +109,7 @@ type ChildType = {
         plus?: boolean
         arrow_thin?: boolean
         is_showed?: boolean
+        classname?: string
```
Comment (mohammad-rework) [inline src/components/elements/accordion.tsx:112]: It's fine to use `className` as `props` and we don't need to rename it to `classname`.
But if we are already using `className` for another purpose then we can use other names such as `styleName` or `childStyle`. @prince-deriv can suggest the best approach in here.

Comment (prince-deriv) [reply src/components/elements/accordion.tsx:112]: `className` as props should work specially if it's our custom components.

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**